diff --git a/node_modules/onnxruntime-react-native/android/build.gradle b/node_modules/onnxruntime-react-native/android/build.gradle
index 41b4359..0c39494 100644
--- a/node_modules/onnxruntime-react-native/android/build.gradle
+++ b/node_modules/onnxruntime-react-native/android/build.gradle
@@ -247,7 +247,8 @@ dependencies {
     extractLibs "com.microsoft.onnxruntime:onnxruntime-android:latest.integration@aar"
   }
   
-  if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
+  def rnVersionParts = REACT_NATIVE_VERSION.split("\\.")
+  if (rnVersionParts[0].toInteger() == 0 && rnVersionParts[1].toInteger() < 71) {
     extractLibs "com.facebook.fbjni:fbjni:+:headers"
     extractLibs "com.facebook.fbjni:fbjni:+"
   }
diff --git a/node_modules/onnxruntime-react-native/unimodule.json b/node_modules/onnxruntime-react-native/unimodule.json
deleted file mode 100644
index c2bd3c0..0000000
--- a/node_modules/onnxruntime-react-native/unimodule.json
+++ /dev/null
@@ -1,4 +0,0 @@
-{
-  "name": "onnxruntime-react-native",
-  "platforms": ["ios", "android"]
-}
